### DISCUSSION TITLE
Release Management

### DISCUSSION THREAD

**Discussion by RamiAwar** [knqyf263/pet#254] [Q&A] [answered]
Hey @knqyf263! How do we want to manage releases?

There were a bunch of high prio bug fixes that went in this past week, I think a minor release would be nice now.

I can take care of drafting the release notes, but not sure how you want to manage the builds + releases.

I was using goreleaser for Superpet, makes things very easy, I can also set it up for Pet.

And do we want to share access to the homebrew tap?

**Comment by knqyf263** [comment, marked as answer]
It is already set up. Pushing a tag triggers a new release.
https://github.com/knqyf263/pet/blob/main/.goreleaser.yml

>And do we want to share access to the homebrew tap?

GitHub Actions updates the homebrew tap.
https://github.com/knqyf263/pet/blob/e1b209b6d3e79765c9c48bc95154c2218ff6cf03/.goreleaser.yml#L6-L22

**Reply by RamiAwar** [reply]
Will try it out today, thanks!

**Reply by knqyf263** [reply]
I believe all you need to do is push a new tag 😉 Please let me know if it doesn't work.

**Reply by RamiAwar** [reply]
Thanks, that worked 😄
